pkg/proxy/proxyinfo.go

```diff
@@ -54,22 +70,30 @@ func GetProxyInfo(kubeconfig, configContext, revision, istioNamespace string) (*
 			pi = append(pi, istioVersion.ProxyInfo{
 				ID:           ss.ProxyID,
 				IstioVersion: ss.SyncStatus.IstioVersion,
+				Type:         ConvertNodeType(ss.ProxyType),
 			})
 		}
 	}
 
 	return &pi, nil
 }
 
-// GetIDsFromProxyInfo is a helper function to retrieve list of IDs from Proxy.
+// GetIDsFromProxyInfo is a helper function to retrieve list of IDs from Proxy. Which contains only the data panel.
```

Comment:
Ingressgateways are data plane proxies as well, i think this additional comment is misleading for the function